src/utils/serverRunner.js

```diff
@@ -1,27 +1,44 @@
 'use strict';
 const co = require('co');
 const throng = require('throng');
-const gracefulShutdown = require('http-graceful-shutdown');
+const gracefulShutdown = require('./gracefulShutdown');
 
-const logger = require('../logger').getLogger(module);
-const isDevelopment = process.env.NODE_ENV === 'development';
+const Logger = require('../logger');
+const logger = Logger.getLogger(module);
+const isDevelopment = (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test');
```

Comment:
For the tests ;)
Running a the cluster in tests wasn't simple so the units tests are testing this without the cluster part.